model_compression_toolkit/core/common/quantization/core_config.py

```diff
@@ -22,20 +23,23 @@ class CoreConfig:
     A class to hold the configurations classes of the MCT-core.
     """
     def __init__(self,
-                 quantization_config: QuantizationConfig = QuantizationConfig(),
+                 quantization_config: QuantizationConfig = None,
                  mixed_precision_config: MixedPrecisionQuantizationConfig = None,
-                 debug_config: DebugConfig = DebugConfig()
+                 bit_width_config: BitWidthConfig = None,
+                 debug_config: DebugConfig = None
                  ):
         """
 
         Args:
             quantization_config (QuantizationConfig): Config for quantization.
             mixed_precision_config (MixedPrecisionQuantizationConfig): Config for mixed precision quantization.
             If None, a default MixedPrecisionQuantizationConfig is used.
+            bit_width_config (BitWidthConfig): Config for bit-width selection.
```

Comment:
"for **manual** bit-width selection"
otherwise, it sounds like this is the proper way to set a bit-width, which is not